sdk/servicebus/service-bus/src/core/managementClient.ts

```diff
@@ -530,10 +554,10 @@ export class ManagementClient extends LinkEntity {
    * LockDuration set on the Entity.
    *
    * @param {string} lockToken Lock token of the message
-   * @param {SendRequestOptions} [options] Options that can be set while sending the request.
+   * @param {SendManagementRequestOptions} [options] Options that can be set while sending the request.
    * @returns {Promise<Date>} Promise<Date> New lock token expiry date and time in UTC format.
    */
-  async renewLock(lockToken: string, options?: SendRequestOptions): Promise<Date> {
+  async renewLock(lockToken: string, options?: SendManagementRequestOptions): Promise<Date> {
     throwErrorIfConnectionClosed(this._context.namespace);
     if (!options) options = {};
```

Comment:
Mentioning this since the signature is being touched, but this could be simplified to:
```suggestion
  async renewLock(lockToken: string, options: SendManagementRequestOptions = {}): Promise<Date> {
    throwErrorIfConnectionClosed(this._context.namespace);
```
Then TypeScript knows that options is always defined vs currently where it thinks it could be undefined and forces you to use the `?` operator.

Documentation still views `options` as optional in this case since that's implied with the default parameter.

Reply:
This would work if the options was the only optional argument or if the optional arguments coming before it had default values too. The peek() & receiveDeferredMessages() operations have multiple optional arguments with no default. So, I didn't want to get into this pattern just ye

Reply:
Fair enough!